augur/clades.py

```diff
@@ -181,6 +202,8 @@ def register_arguments(parser):
     parser.add_argument('--mutations', nargs='+', help='JSON(s) containing ancestral and tip nucleotide and/or amino-acid mutations ')
     parser.add_argument('--reference', nargs='+', help='fasta files containing reference and tip nucleotide and/or amino-acid sequences ')
     parser.add_argument('--clades', type=str, help='TSV file containing clade definitions by amino-acid')
+    parser.add_argument('--trait-name', type=str, help='name to use to store clade membership (set for every node belonging to the clade)')
```

Comment:
We've been using `--attribute-name` for this type of argument in [augur](https://github.com/nextstrain/augur/blob/d1f2fd55df698b219d80ccf42940faf3dbd08e5e/augur/distance.py#L631) and [elsewhere in the ncov workflow](https://github.com/nextstrain/ncov/blob/bbaf167d5b0dd8bdeff3f4353a6b113556e4caa6/scripts/make_pangolin_node_data.py#L21). It's intentionally pretty generic, but it would avoid confusion here with the existing meaning of "trait" from `augur traits`.

Reply:
Done 👍